### ISSUE TITLE
Using v1.5.0 seems to erase `R` from the `PATH`

### ISSUE THREAD

**Issue by lcolladotor** [hrbrmstr/RSwitch#6]
Hi,

I'm not sure if this is the best place to report this, if not, please let me know! Maybe you prefer https://git.rud.is/hrbrmstr/RSwitch/issues for example.

I'm a new `hrbrmstr/RSwitch` user so maybe I missed an installation step. Using `RSwitch` version 1.5.0 apparently is not working for me as it's no longer finding the soft link `/usr/local/bin/R` though `which R` on the terminal after I use `RSwitch` v1.5.0.

First, I can see that it detects the correct version of R:

<img width="276" alt="Screen Shot 2019-09-20 at 10 34 22 AM" src="https://user-images.githubusercontent.com/2288213/65335927-60aaea00-db93-11e9-99ba-1f555f4f5d5c.png">

On the terminal (`iTerm2` but also happens in the `Terminal` app) I can see the soft link and R is available:

```bash
10:34 $ which R
/usr/local/bin/R
10:34 $ ls -lh /usr/local/bin/R
lrwxr-xr-x 1 root admin 47 Jul 27 09:06 /usr/local/bin/R -> /Library/Frameworks/R.framework/Resources/bin/R
```

If I then click on `3.6 (3.6.1)` in `RSwitch` v1.5.0, I can see that `RSwitch` doesn't seem to know which R version it's point to anymore (image below) although I did get the "Success" notification with the correct version number.

<img width="276" alt="Screen Shot 2019-09-20 at 10 34 42 AM" src="https://user-images.githubusercontent.com/2288213/65336072-a5cf1c00-db93-11e9-92b0-86812e5ea7d9.png">

I can then confirm that R is no longer found through `which` despite the soft link being there. Hm...

```bash
## Click on Rswitch 
10:34 $ which R
10:34 $
10:34 $ ls -lh /usr/local/bin/R
lrwxr-xr-x 1 root admin 47 Jul 27 09:06 /usr/local/bin/R -> /Library/Frameworks/R.framework/Resources/bin/R
```

This affects launching from `RStudio` for example since I get the `Unable to locate R binary by scanning standard locations` error message from `RStudio`.

If I then open `RSwitch` version 1.2 from [mac.r-project.org/](http://mac.r-project.org/) (from the `RSwitch-1.2.dmg` file, so it's not in `Applications` and wasn't running while `RSwitch` v1.5.0 was) then I can change R versions again.

Thank you for your help!

Best,
Leo

## Mac info

If you need more information for reproducing this issue, please let me know. I'm not sure what else to provide.

* macOS version 10.14.6
* `RSwitch` version 1.5.0

**Comment by hrbrmstr** [comment]
Well met, Leo!

Thanks for taking RSwitch for a spin.

This is a fine place to report a bug and this seems to be a big one.

I’m a few thousand feet in the air (heading back from DSC 2019) so I won’t
be able to poke at this until tonight or tomorrow but I wanted to shoot a
note to let you know I received it.

-boB

**Comment by hrbrmstr** [comment]
And it is definitely a bug due to a "clever" code change made after having already tested it #sigh.

Many thanks for flagging it!

Tis now fixed in 1.5.1. The Check for updates… should take you to the download page but it's also a full release: https://github.com/hrbrmstr/RSwitch/releases/tag/v1.5.1

**Comment by lcolladotor** [comment]
Awesome, thanks for making this fix! I can confirm that `RSwitch` version 1.5.1 works for me =) Thank you again! 🙌🏽
